(IAC-1008) - Removal of Inappropriate Terminology (#148)
puppetlabs/puppetlabs-satellite_pe_tools: README.md

   `~/ssl-build/puppet.example.com/etc/pki/katello-certs-tools/certs/puppet.example.com-puppet-client.crt`
     `~/ssl-build/puppet.example.com/etc/pki/katello-certs-tools/private/puppet.example.com-puppet-client.key`

> Copy the files to `/etc/puppetlabs/puppet/ssl/satellite` (on PE >= 2015.x) or `/etc/puppet/ssl/satellite` (PE 3.x) on your server.

Reviewer: Is PE 3.x still relevant?

Author: There are some people that still use it, but removed support for it in the newest versions of this module so I feel confident in removing it.

chown pe-puppet /etc/puppetlabs/puppet/ssl/satellite/puppet.example.com-puppet-client.key
```

> f. In the Satellite UI, go to *Administer -> Settings -> Auth* and set the `restrict_registered_puppetservers` parameter to `true`. Additionally, add your Puppet server's FQDN to the `trusted_puppetserver_hosts` array on the same page; for example, `[puppet.example.com]`.

Author: Have swapped this back

> f. In the Satellite UI, go to *Administer -> Settings -> Auth* and set the `restrict_registered_puppetmasters` parameter to `true`. Additionally, add your Puppet server's FQDN to the `trusted_puppetserver_hosts` array on the same page; for example, `[puppet.example.com]`.

Reviewer: `trusted_puppetserver_hosts` doesn't exist. Also, `trusted_puppetmaster_hosts` has been renamed to `trusted_hosts` in 1.18. That should cover all supported Satellite versions.

Author: have changed it to refer to trusted_hosts

> On Satellite 6.2 (and since Foreman 1.8.0) the `restrict_registered_puppetmasters` setting has been renamed to `restrict_registered_smart_proxies` (labelled "Restrict registered capsules"). `trusted_puppetserver_hosts` has been given the label "Trusted puppetserver hosts" in the UX. You can see the actual setting names by mousing over the label.

Reviewer: Satellite 6.5 and older are EOL. That means you should use `restrict_registered_smart_proxies` and don't really mention `restrict_registered_puppetmasters` at all.

Author: Have cut out the bit about 6.2 and updated it to refer to trusted_hosts

g. Set the `ssl_cert` and `ssl_key` parameters in your `satellite_pe_tools` class to the location on your Puppet server of the two files respectively.

> If you do not want the Satellite server to verify the Puppet server identity, then in the Satellite UI, go to *Administer -> Settings -> Auth* and set the `restrict_registered_smart_proxis` parameter to `false`.

Reviewer: ```suggestion
If you do not want the Satellite server to verify the Puppet server identity, then in the Satellite UI, go to *Administer -> Settings -> Auth* and set the `restrict_registered_smart_proxies` parameter to `false`.
```

Author: fixed

> If you do not want the Satellite server to verify the Puppet server identity, then in the Satellite UI, go to *Administer -> Settings -> Auth* and set the `restrict_registered_puppetmasters` parameter to `false`.

Reviewer: ```suggestion
If you do not want the Satellite server to verify the Puppet server identity, then in the Satellite UI, go to *Administer -> Settings -> Auth* and set the `restrict_registered_smart_proxies` parameter to `false`.
```

Author: Have updated this

> If you do not want the Satellite server to verify the Puppet server identity, then in the Satellite UI, go to *Administer -> Settings -> Auth* and set the `restrict_registered_puppetservers` parameter to `false`.

Reviewer: `restrict_registered_puppetservers` does not exist.

Author: Have fixed this

}
~~~

> This example tells the server the location of the Satellite server (`https://puppet.example.com`) and instructs it to verify the Satellite server's identity.

Reviewer: Server is an ambiguous name, which is my whole problem with this naming.
```suggestion
This example tells the Puppet server the location of the Satellite server (`https://puppet.example.com`) and instructs it to verify the Satellite server's identity.
```

Author: Have added, the Puppet bit, you have a god point, with the two servers on the one line it can get confusing

> ```

Reviewer: Note: the line above this still uses `katello-service` but that's been replaced with `satellite-maintain restart`. Also don't think you should recommend 1024 bits DH params these days.

Author: Have replaced katello-service with satellite-maintain.
For the 1024 bits have left it as is for now, feel that change should be included as a larger update to the module docs.

Reviewer 2: @david22swan should line 169 also have been modified to specify satellite-maintain instead?

Author: @michaeltlombardi 
Yeh, that change was intentional
